bundles/org.openhab.binding.publictransportswitzerland/src/main/java/org/openhab/binding/publictransportswitzerland/internal/stationboard/PublicTransportSwitzerlandStationboardHandler.java

```diff
@@ -101,6 +104,11 @@ public void initialize() {
             updateStatus(ThingStatus.UNKNOWN);
             startChannelUpdate();
         }
+
+        if (getThing().getChannelsOfGroup(stationboardChannelGroupUID.getId()).isEmpty()) {
+            Channel tsvChannel = ChannelBuilder.create(tsvChannelUID, "String").build();
+            updateThing(editThing().withChannel(tsvChannel).build());
```

Comment:
Is there any reason why you create the Channel dynamically and not in the xml file?

Reply:
I was not aware that I could define channel groups in the xml. I'll look into it.

Reply:
I don't want to push you, but the merge window for 2.5.9 comes closer and closer. If we miss that, the PR must be manually migrated to OH3.

Reply:
Sorry I did not have the time to work on this.
I also wasn't aware of the OH3 release and that there is a merge window (I still wasn't able to figure out when exactly).

I would be very grateful if someone could help me out here.

Reply:
Here is the community post https://community.openhab.org/t/openhab-2-5-x-patch-releases/90248/260

You only need to add a Channel type and use this. I made a suggestion.

After I approved the PR, another maintainer need to do a second review, as this is a new binding.

Reply:
Thank you so much

Reply:
As last thing you only need to remove the dynamic creation of the Channel from the handler above.